packages/shared-ui/src/flowHandler/flows/login/utils.ts

```diff
@@ -68,6 +68,12 @@ const sendEmailLink = async (authService: AuthService, email: string): Promise<F
   return;
 };
 
+export const sendEmailLinkAgain = async (authService: AuthService, email: string): Promise<FlowUpdate> => {
+  const res = await sendEmailLink(authService, email);
```

Comment:
Right now we are not handing the error that can be thrown by sendEmailLink. I think we should handle it exactly as we are doing in "sendEmailOTP".